.github/workflows/golangci-lint.yml

```diff
@@ -18,4 +18,4 @@ jobs:
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v3
         with:
-          version: v1.54.2
+          version: v1.52.2
```

Comment:
Did you want to make this `v1.55.2` instead?

Reply:
duh, thanks